sqlglot/parser.py

```diff
@@ -3880,6 +3881,25 @@ def _parse_merge(self) -> exp.Expression:
             expressions=whens,
         )
 
+    def _parse_pragma(self) -> t.Optional[exp.Expression]:
+        this = self._parse_primary() or self._parse_id_var()
+        schema = None
+        value = None
+        func = None
+
+        while self._match(TokenType.DOT):
+            schema = this
+            this = self._parse_primary() or self._parse_id_var()
+
+        if self._match(TokenType.EQ):
+            self._match(TokenType.EQ)
```

Comment:
We've already consumed the EQ token here, so this seems redundant unless there's a syntax form I missed that expects `==` as a token.

Reply:
yeah, that's a leftover from a previous attempt where I checked the token w/o advancing